### PR TITLE
refactor: update stack with component-rollup-config

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

* use `@ovh-ux/component-rollup-config` to build the package
* remove use of bower
* clean dependencies
* update eslint config
* add missing assets
BREAKING CHANGE: the package is now `@ovh-ux/ng-ovh-line-diagnostics`. The module
is now `ngOvhLineDiagnostics`

0a89def — refactor: update stack with component-rollup-config

# To Do : 

* [x] rename repository to `ng-ovh-line-diagnostics`

https://github.com/ovh-ux/manager/issues/188

### CODE REVIEW

File: src/provider.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import angular from 'angular';
+import forIn from 'lodash/forIn';
+import merge from 'lodash/merge';
+
+export default /* @ngInject */ function () {
+  let requestProxy = '$http';
+  let translationPath = '../bower_components/ovh-angular-line-diagnostics/dist';
```
Comment (antleblanc) [inline src/provider.js]: No more needed?

**Review by antleblanc — CHANGES_REQUESTED** [review]
Well done!

Just a quick note regarding some parts that still mention `bower`.

File: src/provider.js (hunk 1 above)
Comment (cbourgois) [reply src/provider.js]: Nope, I have to remove it.

Comment (cbourgois) [reply src/provider.js]: Resolved.

File: package.json [hunk 2]
```diff
@@ -1,44 +1,69 @@
...
+    "lint:html": "htmlhint 'src/**/*.html'",
+    "lint:js": "eslint --quiet --fix --format=pretty src",
+    "lint:md": "eslint --quiet --fix --format=pretty --ext .md .",
+    "prepare": "rollup -c",
+    "start": "rollup -c --watch",
+    "test": "yarn run lint",
+    "version": "npx conventional-changelog-cli -p angular -i CHANGELOG.md -s && git add CHANGELOG.md",
+    "postversion": "git push && git push --tags"
+  },
   "dependency": {},
```
Comment (antleblanc) [inline package.json]: This entry could be removed I guess.

**Review by antleblanc — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (cbourgois) [reply package.json]: Removed 👍

**Review by antleblanc — APPROVED** [review]

**Pull request merged**